app/assets/javascripts/app/views/comment_stream_view.js

```diff
@@ -9,10 +9,12 @@ app.views.CommentStream = app.views.Base.extend({
   events: {
     "keydown .comment_box": "keyDownOnCommentBox",
     "submit form": "createComment",
-    "focus .comment_box": "commentTextareaFocused",
-    "click .toggle_post_comments": "expandComments"
+    "click .toggle_post_comments": "expandComments",
+    "click form": "openForm"
   },
 
+  mdEditor: undefined,
```

Comment:
To be explicit that `mdEditor` should be `undefined` when the object is initialized. I still don't really understand big parts of JS so that may be useless. I don't know.